tools/system_libs.py

```diff
@@ -665,7 +664,8 @@ def get_files(self):
       # With the wasm backend these are included in wasm_libc_rt instead
       blacklist += [os.path.basename(f) for f in get_wasm_libc_rt_files()]
     else:
-      blacklist += ['rintf.c']
+      blacklist += ['rintf.c', 'ceil.c', 'ceilf.c', 'floor.c', 'floorf.c',
+                    'fabs.c', 'fabsf.c', 'sqrt.c', 'sqrtf.c']
```

Comment:
Is there a reason these still need blacklisting for asmjs?

Reply:
Blacklisting these will use the JS implementations, which makes sense for asm.js.

Reply:
Won't the clang builtins also to the right thing for asmjs?

Reply:
Won't the clang builtins also to the right thing for asmjs?

Reply:
From what I heard, asmjs is supposed to just use `Math.floor` etc. directly. @kripken would know more.

Reply:
I think those builtins were added for upstream. I don't think fastcomp has them all. It relies on JS library stuff to handle those without intrinsics.